mcsteplogger.sh

```diff
@@ -1,10 +1,10 @@
 package: MCStepLogger
 version: "%(tag_basename)s"
-tag: "v0.2.0"
+tag: "v0.3.0"
 source: https://github.com/AliceO2Group/VMCStepLogger.git
 requires:
   - "GCC-Toolchain:(?!osx)"
-  - ROOT
+  - VMC
```

Comment:
I would prefer to keep the full list of dependencies. It cannot hurt (for people not knowing that VMC will bring in ROOT).

Reply:
ok, sure, I will make the`ROOT` dep explicit again.